pkg/juicefs/mount/builder/job.go

```diff
@@ -135,7 +135,7 @@ func (r *Builder) getDeleteVolumeCmd() string {
 	} else {
 		jfsPath = config.CliPath
 	}
-	return fmt.Sprintf("%s && if [ -d /mnt/jfs/%s ]; then %s rmr /mnt/jfs/%s; fi;", cmd, r.jfsSetting.SubPath, jfsPath, r.jfsSetting.SubPath)
+	return fmt.Sprintf("%s && if [ -d /mnt/jfs/%s ]; then %s rmr /mnt/jfs/%s && %s gc ${metaurl} --delete; fi;", cmd, r.jfsSetting.SubPath, jfsPath, r.jfsSetting.SubPath, jfsPath)
```

Comment:
`gc` is a little different between [ce](https://juicefs.com/docs/zh/community/command_reference#gc) (`juicefs gc $METAURL --delete`) and [ee](https://juicefs.com/docs/zh/cloud/reference/command_reference#gc) (`juicefs gc $VOL_NAME --delete`).

Reply:
I fixed it.
I can't test it under ``ee``, but it works well on ``ce``.